app/views/planeadores_de_eventos/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+
+<%= form_with(model: planeadores_de_evento) do |form| %>
+  <% if planeadores_de_evento.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(planeadores_de_evento.errors.count, "error") %> prohibited this planeadores_de_evento from being saved:</h2>
+
+      <ul>
+        <% planeadores_de_evento.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :titulo, style: "display: block" %>
+    <%= form.text_field :titulo, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :descripcion, style: "display: block" %>
+    <%= form.text_area :descripcion, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :fecha, style: "display: block" %>
+    <%= form.datetime_field :fecha, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :ubicacion, style: "display: block" %>
+    <%= form.text_area :ubicacion, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.label :costo, style: "display: block" %>
+    <%= form.text_field :costo, class:"form-control" %>
+  </div>
+
+  <div>
+    <%= form.submit class:"btn btn-success", style:"margin-top:30px" %>
+  </div>
+<% end %>
```

Comment:
Add a new line at the end of the file 
Also youre still using scaffold 
This isnt part of the scope of the pr youre describing